mappings/net/minecraft/util/shape/DoubleListPair.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS cra net/minecraft/util/shape/DoubleListPair
+	CLASS net/minecraft/util/shape/DoubleListMerger$a SectionPairPredicate
+		METHOD merge test (III)Z
```

Comment:
Why not keep the Mojang name here?

Reply:
Didn't realize it was a Mojang name (using Knit so there's no highlighting). It seems weird to call the method of a predicate `merge` though...

Reply:
I haven't read through the code much, but it does seem to be used specifically for merging voxel shapes.

Reply:
It's passed as a parameter to the `forAllOverlappingSections`, which returns true if the predicate is true for all pairs of sections in both lists that overlap. It's used by both the shape comparison and shape merging code, but it still seems weird for a boolean-valued method with no side effects to be called `merge`.

Reply:
See `BitSetVoxelSet#combine`: the `SectionPairPredicate` passed to `forAllOverlappingSections` produces side effects through its implementation. Imo, `merge` does make some sense here.
Also, not necessarily a good name, but I imagine `forAllOverlappingSections` would be `merge` too